src/features/dashboard/components/AppForm/index.tsx

```diff
@@ -250,7 +250,7 @@ const AppForm = ({
                   type='text'
                   placeholder=' '
                 />
-                <label htmlFor='app_redirect_uri'>Authorisation URL (optional)</label>
+                <label htmlFor='app_redirect_uri'>Redirect URL (optional)</label>
```

Comment:
@ahmed-deriv , we have removed the optional in our new dashboard as when the markup is nonzero the redirect url is required and when it's 0 it's optional. So do we want to remove it here as well?

Reply:
I think currently we don't have this validation on production so lets keep it the way it was. Once we sync the branch it auto remove the `optional` text.